test-utils/runtime/src/genesismap.rs

```diff
@@ -17,75 +17,116 @@
 
 //! Tool for creating the genesis block.
 
-use super::{system, wasm_binary_unwrap, AccountId, AuthorityId, Runtime};
-use codec::{Encode, Joiner, KeyedVec};
-use frame_support::traits::GenesisBuild;
+use super::{
+	currency, substrate_test_pallet, wasm_binary_unwrap, AccountId, AuthorityId, Balance,
+	GenesisConfig,
+};
+use codec::{Encode, Joiner};
```

Comment:
```suggestion
use codec::Encode;
```
you don't need this if you apply the change below